viewer/js/config/viewer.js

```diff
@@ -295,7 +295,7 @@ define([
                     visible: true,
                     outFields: ['*'],
                     imageParameters: buildImageParameters({
-                        layerIds: [0, 7],
+                        layerIds: [0, 6, 7, 8],
```

Comment:
All these changes to layerIds were done under issue #23 and merged to publicSiteMap branch in 68cfffb. I don't know why this pull request should be trying to re-apply these changes.